pkg/config/config.go

```diff
@@ -248,10 +248,14 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("confd_path", defaultConfdPath)
 	config.BindEnvAndSetDefault("additional_checksd", defaultAdditionalChecksPath)
 	config.BindEnvAndSetDefault("jmx_log_file", "")
+	config.BindEnvAndSetDefault("dogstatsd_log_file", "")
 	config.BindEnvAndSetDefault("log_payloads", false)
 	config.BindEnvAndSetDefault("log_file", "")
 	config.BindEnvAndSetDefault("log_file_max_size", "10Mb")
 	config.BindEnvAndSetDefault("log_file_max_rolls", 1)
+	config.BindEnvAndSetDefault("dogstatsd_stats_logging", true)
+	config.BindEnvAndSetDefault("dogstatsd_log_file_max_rolls", 3)
+	config.BindEnvAndSetDefault("dogstatsd_log_file_max_size", "10Mb")
```

Comment:
Would it make sense to keep these settings together with the rest of dogstatsd options? (Somewhere around https://github.com/datadog/datadog-agent/blob/563ac9ff40cb5755a71728f4bd768a738d6c8f97/pkg/config/config.go#L523)